### PR TITLE
Add simplest way to use Stylelint

### PR DESCRIPTION
**Pull request by ai**
## Description
Addresses issue #44

It is only initial step.

## List of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [ ] My code is tested.
- [x] I want my code added to WP Rig.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -168,13 +169,6 @@ export function styles() {
 
 	return gulp.src(paths.styles.src)
 	.pipe(print())
-	.pipe(phpcs({
-		bin: 'vendor/bin/phpcs',
-		standard: 'WordPress',
-		warningSeverity: 0
-	}))
-	// Log all problems that was found
-	.pipe(phpcs.reporter('log'))
```
Comment (ntwb) [inline gulpfile.babel.js]: This shouldn't be removed @ai

Comment (ai) [reply gulpfile.babel.js]: Oops. I thought it is a way to run CSS Lint

**Comment by mor10** [comment]
Once #47 is merged with the 1.1 branch, we'll expand on this to get it working. Looping in @ataylorme for reference.

**Comment by mor10** [comment]
Closing in favor of #56 which extends this proposal and merges it with the refactored Gulp approach.

**Pull request closed without merging**